src/main/java/com/netgrif/application/engine/configuration/authentication/providers/ldap/NetgrifLdapAuthenticationProvider.java

```diff
@@ -132,6 +133,11 @@ public void setUseAuthenticationRequestCredentials(boolean useAuthenticationRequ
 
     @Override
     public Authentication authenticate(Authentication authentication) throws AuthenticationException {
+        WebAuthenticationDetails details = (WebAuthenticationDetails) authentication.getDetails();
+        String key = details.getRemoteAddress();
+        if (key == null) {
+            throw new BadCredentialsException("Bad credentials");
```

Comment:
The message in the Exception here is inconsistent with the message in the `Basic` auth provider.

Reply:
basic auth contains information using MessageSourceAccessor messages = SpringSecurityMessageSource.getAccessor();